tests/unit/pyrflx_test.py

```diff
@@ -1140,30 +1140,116 @@ def test_checksum_value_range(no_conditionals_message: Message) -> None:
     assert not msg._is_checksum_settable(msg._checksums["Checksum"])
 
 
-def test_refinement_with_checksum() -> None:
+@pytest.fixture(name="tlv_checksum_function")
+def fixture_tlv_checksum_function() -> ty.Callable:
     def tlv_checksum(_: bytes, **__: object) -> int:
         return 0
 
+    return tlv_checksum
+
+
+@pytest.fixture(name="refinement_checksum_function")
+def fixture_refinement_checksum_function() -> ty.Callable:
     def msg_checksum(_: bytes, **__: object) -> int:
         return 0xFF
```

Comment:
Is there an advantage in having fixtures instead of just functions? The naming is also a bit odd. `tlv_checksum_function` gives the impression the function is specific to TLV, but it is always returning zero.

Reply:
Renamed the functions and removed fixture in dce0b75